test/e2e/common/e2e_config/e2e_config.go

```diff
@@ -30,7 +33,7 @@ type E2EConfig struct {
 		Duration         string   `yaml:"duration" env-default:"10m"`
 		LoadFactor       int      `yaml:"loadFactor" env-default:"10"`
 		Protocols        []string `yaml:"protocols" env-default:"nvmf,iscsi"`
-		FioFixedDuration int      `yaml:"fioFixedDuration" env-default:"60"`
+		FioStartDelay    int      `yaml:"fioStartDelay" env-default:"60"`
```

Comment:
Perhaps this field name should include an indication of the unit?  It's not clear to me on sight, whether the default value of "60" should infer minutes or seconds

Reply:
I will add comments describing units